learning_journal_db/templates/layout.jinja2

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html>
+
+    <head>
+
+      <meta name="viewport" content="width=device-width, initial-scale=1">
+        <link href='https://fonts.googleapis.com/css?family=PT+Serif|Montserrat' rel='stylesheet' type='text/css'>
+        <link rel="stylesheet" href="../static/base.css">
+
+    </head>
+
+    <body>
+
+        <header>
+              <h1>Learning Journal<h1>
```

Comment:
You don't have a closing </h1> tag here. You probably meant for that to be here.